### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by peternepp** [fabiangeisler/fg-tools#3] [General]
I've figured out some issues during the tests for Maya 2020. Any interest writing them down and fixing them once in a while?

**Comment by fabiangeisler** [comment]
Sure! Thanks for the effort and no rush! 🙂

**Reply by peternepp** [reply]
Alright. I've to figure out how to handle issues across forks: (Pipeline Developer through and through 😉 )

Maybe we collect issues in your repository first and if I'm going to fix one myself, making a transfer of this particular issue to my fork. I don't know if this is going to work yet. I know there's a 'Transfer Issue' feature on Github, but it might require a dedicated permission to do that. Do you have any experience with it?

Any other suggestions?

**Reply by fabiangeisler** [reply]
I think Issues cannot be transferred across forks. Don't know either what is the best workflow here 🤔 .

**Reply by peternepp** [reply]
Hmm, I'm going to check this out. 👍

**Reply by peternepp** [reply]
I'm afraid you are right. It's not possible to transfer issues to a fork easily. But there are some external tools to manage a scenario like this:

* [lagerfeuer/github-issue-clone: Clone Github issue from one repository to another](https://github.com/lagerfeuer/github-issue-clone)
* [soichisumi/issue-copy-action: GitHub Action to copy issues to another repository](https://github.com/soichisumi/issue-copy-action)
* [vivekseth/github-copy-issues](https://github.com/vivekseth/github-copy-issues)

I don't know if it's getting to complicated, but I'm curious about such workflows though.

But let's keep it simple, let's collect issue here and if I'm going to work on an issue I can manage this quite simple with my fork. Maybe there's no need to work with issues there, just reference the original issue number with the branch, PRs ... or something.

Alternatively, being a collaborator with the original repository should work as well, but I actually prefer keeping a forking workflow for now.
